onnxruntime/core/providers/dml/DmlExecutionProvider/src/ExecutionProvider.cpp

```diff
@@ -1181,14 +1189,62 @@ namespace Dml
         return onnxruntime::common::Status::OK();
     }
 
+    void ExecutionProviderImpl::AppendCapturedGraph(int annotationId, std::unique_ptr<DmlReusedCommandListState> capturedGraph)
+    {
+        m_capturedGraphs[annotationId].push_back(std::move(capturedGraph));
+    }
+
+    onnxruntime::common::Status ExecutionProviderImpl::ReplayGraph(int graph_annotation_id)
+    {
+        for (auto& capturedGraph : m_capturedGraphs[graph_annotation_id])
```

Comment:
```suggestion
        for (auto& capturedGraph : m_capturedGraphs[graphAnnotationId])
```
Intra-file self consistency.

And `dml_graph_annotation_id` too below.